### PR TITLE
[E2E] Add Stream tests added

### PR DESCRIPTION
**Pull request by elenaNaboko**

### CODE REVIEW

File: tests/e2e/pageObjects/browser-page.ts [hunk 1]
```diff
@@ -159,6 +164,23 @@ export class BrowserPage {
...
+     * Common part for Add any new key
+     * @param keyName The name of the key
+     * @param TTL The Time to live value of the key
+     */
+    async commonAddNewKey(keyName: string, TTL?: string): Promise<void> {
+        await common.waitForElementNotVisible(this.progressLine);
+        await t.click(this.plusAddKeyButton);
+        await t.click(this.addKeyNameInput);
+        await t.typeText(this.addKeyNameInput, keyName);
+        if (TTL) {
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/browser-page.ts:177]: Better: `if (TTL !== undefined)`

File: tests/e2e/pageObjects/browser-page.ts [hunk 2]
```diff
@@ -282,6 +304,24 @@ export class BrowserPage {
         await t.click(this.addKeyButton);
     }
 
+    /**
+     * Adding a new Stream key
+     * @param keyName The name of the key
+     * @param TTL The Time to live value of the key
+     * @param field The field name of the key
+     * @param value The value of the key
```
Comment (ofersteinberg) [inline tests/e2e/pageObjects/browser-page.ts:312]: It is not in the same order as the function

File: tests/e2e/pageObjects/browser-page.ts (hunk 1 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/browser-page.ts]: fixed

File: tests/e2e/pageObjects/browser-page.ts (hunk 2 above)
Comment (elenaNaboko) [reply tests/e2e/pageObjects/browser-page.ts]: fixed

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**